.github/workflows/tests.yml

```diff
@@ -72,3 +72,23 @@ jobs:
     - name: "Run HLS test suite"
       run: make -j `nproc` -O run
       shell: bash
+
+    - name: "Get the contents of the cycle-diff.log if it exists"
+      id: get-cycles-log
+      if: ${{ hashFiles('build/cycle-diff.log') != '' }}
+      run: |
+        echo "::set-output name=cycles::$(cat build/cycle-diff.log)"
```

Comment:
This is the old style of setting variables. The new way is
`echo "cycles=$(cat build/cycle-diff.log)" >> $GITHUB_OUTPUT`
But this seems to cause the contents of the file to be interpreted and fails.

Reply:
The error reported with the new way of setting outputs.

Run echo "cycles=$(cat build/cycle-diff.log)" >> $GITHUB_OUTPUT
  echo "cycles=$(cat build/cycle-diff.log)" >> $GITHUB_OUTPUT
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    JLM_ROOT_DIR: /tmp/jlm
Error: Unable to process file command 'output' successfully.
Error: Invalid format '  Golden cycle time: 52'

Reply:
The old style seems to only set the variable to the first line of the file.